packages/by-name/microsoft/genpolicy/package.nix

```diff
@@ -57,8 +57,12 @@ rustPlatform.buildRustPackage rec {
       ./0006-genpolicy-support-HostToContainer-mount-propagation.patch
       # This patch is a port of https://github.com/kata-containers/kata-containers/pull/10136/files
       # to Microsofts genpolicy.
-      # TODO(miampf): remove when picked up by microsoft/kata-containers fork.
+      # TODO(miampf):
       ./0007-genpolicy-support-for-VOLUME-definition-in-container.patch
+
+      # Simple genpolicy logging patch to include the image reference in case of authentication failure
+      # Not merged, TODO(jmxnzo): remove when picked up by microsoft/kata-containers fork
```

Comment:
How should this be picked up by the microsoft fork if we aren't proposing it upstream? Or what exactly must be picked up so this can be dropped?

Reply:
It was by accident, when copy-pasting. thanks for pointing out!